vizro-core/examples/features/app.py

```diff
@@ -14,6 +15,9 @@
 from vizro.tables import dash_data_table
 
 iris = px.data.iris()
+gapminder = px.data.gapminder()
+tips = px.data.tips()
+tips["smoker"] = np.where(tips["smoker"] == "Yes", 1, 0)
```

Comment:
```suggestion
tips["smoker"] = tips["smoker"] == "Yes"
```
Would work ok depending on how we and plotly handle `True/False` in pandas series. So don't just accept this suggestion without checking it because it might break everything 😅

Reply:
Also see later comment on whether we want to be filtering on this column at all.

Reply:
I removed it entirely 👍